src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -567,5 +568,11 @@
 			"\tif the clicked button is enchantment option 1:",
 			"\t\tset the applied enchantments to sharpness 10 and unbreaking 10")
 		.since("2.5");
+		if(Skript.classExists("org.bukkit.event.block.BlockFertilizeEvent"))
```

Comment:
Since we're checking if this class exists and if im not mistaken its introduced in 1.16.1, it should be added here which version it works on for the docs

Reply:
The fertilize event was added in 1.13, but yeah I'll add it there

Reply:
Oh sorry I was thinking about the fertilize block method that was recently added. But yes, 1.13 should be added.